templates/engage/unlisted/custom-ubuntu-pro-image-on-azure.html

```diff
@@ -0,0 +1,65 @@
+{% extends "engage/base_engage.html" %}
+
+{% block title %}Using Shell's Ubuntu Pro image on Azure{% endblock %}
+{% block meta_description %}Get started with custom images of Ubuntu Pro with Support on Azure{% endblock %}
+
+{% block head_extra %}
+<meta name="robots" content="noindex, nofollow" />
```

Comment:
I am slightly worried that someone might create a page here in the future and omit this piece which is the important contract with the unlisted page. I wonder if it's better to create a new base_engage that enforces this?

Reply:
That's a good point/something I wondered - I originally went with adding this to both templates, thinking in future people would just use them as references for other pages and copy that block over, and we then wouldn't need to create a new base, but maybe the trade-off of another template is worth it if it makes it less error-prone.